esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -919,15 +964,29 @@ void EsdlServiceImpl::handleServiceRequest(IEspContext &context,
         }
     }
 
-    context.addTraceSummaryTimeStamp(LogNormal, "srt-resLogging");
-    handleResultLogging(context, scriptContext, tgtctx.get(), req, soapmsg.str(), origResp.str(), out.str(), logdata.str());
-    context.addTraceSummaryTimeStamp(LogNormal, "end-resLogging");
     ESPLOG(LogMax,"Customer Response: %s", out.str());
 }
 
-bool EsdlServiceImpl::handleResultLogging(IEspContext &espcontext, IEsdlScriptContext *scriptContext, IPropertyTree * reqcontext, IPropertyTree * request,const char *rawreq, const char * rawresp, const char * finalresp, const char * logdata)
+bool EsdlServiceImpl::handleResultLogging(IEspContext &espcontext, IEsdlScriptContext *scriptContext, IEsdlDefService &srvdef, IEsdlDefMethod &mthdef, IPropertyTree * reqcontext, IPropertyTree * request,const char *rawreq, const char * rawresp, const char * finalresp, const char * logdata)
 {
+    StringBuffer temp;
+    if (scriptContext)
+    {
+        IEsdlTransformSet *servicePLTs = m_transforms->queryMethodEntryPoint("", ESDLScriptEntryPoint_PreLogging);
+        IEsdlTransformSet *methodPLTs = m_transforms->queryMethodEntryPoint(mthdef.queryName(), ESDLScriptEntryPoint_PreLogging);
+
+        scriptContext->appendContent(ESDLScriptCtxSection_LogData, "LogDataset", logdata);
```

Comment:
I think the second parameter may be misspelled. Current scripts expect the element to be named "LogDatasets". Although it is redundant the current structure is UpdateLogRequest/LogDatasets/LogDatasets/Dataset[@name="..."]

Reply:
Agreed.  Will change.